### PR TITLE
use blocks 0 through 199 for sqlite and pg test databases

### PR DESCRIPTION
**Pull request by buck54321**
Changes the compressed files and queries to create the test databases using the first 200 blocks.

### CODE REVIEW

File: sqlitedb/queries.md [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
-    CREATE TABLE test_dcrdata_stakeinfo_extended AS SELECT * FROM dcrdata_stakeinfo_extended WHERE height > 131800 LIMIT 200;
+   CREATE TABLE test_dcrdata_block_summary AS SELECT * FROM dcrdata_block_summary WHERE height < 200;
+   CREATE TABLE test_dcrdata_stakeinfo_extended AS SELECT * FROM dcrdata_stakeinfo_extended WHERE height < 200;
 ```
 
 - Create dump files with the newly created tables' data and dump data into `data_1.sql` and `data_2.sql` as shown.
 ```Shell
     .output data_1.sql
-    .dump test_dcrdata_block_summary;
+    .dump test_dcrdata_block_summary
```
Comment (buck54321) [inline sqlitedb/queries.md:21]: I may have an outdated version of `sqlite3` or something, but this trailing semi-colon was causing problems for me.

**Review by chappjc — COMMENTED** [review]
Looks good to me.  Although the data has no stake transactions so if there is a need for that, more data would need to be generated.  Since the repo size isn't getting any smaller, perhaps this PR can generate new archive files names (e.g. pgsql-blocks0to199.tar.xz) instead of changing the existing ones.  I believe the existing archives can even be renamed without increasing the repo size much via git mv.

**Comment by chappjc** [comment]
Bump on my comment above.  Are stake transactions not involved in [these charts tests](https://github.com/decred/dcrdata/pull/1274)?

**Comment by buck54321** [comment]
As of right now, the tests I need the zero-based DB for don't need stake transactions. If we need them on a zero-based DB in the future, I guess we'll just need to grab > 4096 blocks. 

I'll update this today to keep both data sets. On the dcrdata side, I'll add a flag to `dbload` and the test script to select which version to use, with zero-based DB as the default.

**Comment by buck54321** [comment]
This actually needs updated with new db schema. Sort of outgrowing the title, but if nobody minds, I'll do the updates in this PR.

**Comment by chappjc** [comment]
Ah, right.  Yeah, this PR is all good for that too

**Review by chappjc — APPROVED** [review]
